homeassistant/components/generic_thermostat/climate.py

```diff
@@ -272,6 +273,15 @@ def precision(self):
             return self._temp_precision
         return super().precision
 
+    @property
+    def target_temperature_step(self) -> Optional[float]:
+        """Return the supported step of target temperature."""
+        # Since this integration does not yet have a step size parameter
+        # we have to re-use the precision as the step size for now.
+        if self._temp_precision is not None:
+            return self._temp_precision
+        return super().precision
```

Comment:
referring to this https://github.com/home-assistant/core/pull/43744#issuecomment-738773970

should return the parent function instead, otherwise remove ` -> Optional[` since `super().precision` always returns float